otx/algorithms/classification/adapters/mmcls/models/classifiers/byol.py

```diff
@@ -162,17 +162,20 @@ def train_step(self, data: Dict[str, Any], optimizer):
         return outputs
 
     def val_step(self, *args):
-        """Disable validation step during self-supervised learning."""
+        """Disenable validation step during self-supervised learning."""
```

Comment:
If functional changes are required, I'll update it with that process.
If not, it can be processed with the refactoring to be processed later together.

Reply:
Done.